crates/forge-runner/Cargo.toml

```diff
@@ -15,6 +15,9 @@ cairo-lang-utils.workspace = true
 cairo-lang-starknet.workspace = true
 cairo-lang-compiler.workspace = true
 cairo-lang-filesystem.workspace = true
+cairo-lang-sierra-type-size = "2.3.0"
+cairo-lang-sierra-gas = "2.3.0"
+cairo-lang-sierra-ap-change = "2.3.0"
```

Comment:
Putting those in the top level Cargo.toml and inheriting from workspace is better, otherwise bumping will be a nightmare